db/migrate/20151127222755_convert_raw_test_runs_payload_to_jsonb.rb

```diff
@@ -0,0 +1,9 @@
+class ConvertRawTestRunsPayloadToJsonb < ActiveRecord::Migration
+  def up
+    execute 'ALTER TABLE raw_test_runs ALTER COLUMN payload SET DATA TYPE jsonb USING payload::jsonb;'
```

Comment:
Line is too long. [102/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.